src/app/components/OptimizelyPageMetrics/experimentsForPageMetrics.ts

```diff
@@ -11,9 +11,9 @@ type ExperimentsForPageTypeMetrics = {
 
 const experimentsForPageMetrics: ExperimentsForPageTypeMetrics = [
   {
-    // EXPERIMENT: Read Time
+    // EXPERIMENT: Read Time & Continue Reading button for articles
     pageType: ARTICLE_PAGE,
-    activeExperiments: ['newswb_ws_article_read_time'],
+    activeExperiments: ['newswb_ws_article_read_time', 'newswb_ws_read_more_b'],
```

Comment:
Should we check if this can now be removed, or is this for the Content team to cleanup?
```suggestion
    activeExperiments: ['newswb_ws_read_more_b'],
```

Reply:
It looks like they have a ticket in their current sprint to do all of their removals, so I will leave it for them to cleanup 🧽

Reply:
OK cool, thanks for checking!